blockchain/blockchain_test.go

```diff
@@ -159,7 +159,7 @@ func TestVerifyBlock(t *testing.T) {
 	t.Run("error when block hash does not match state update's block hash", func(t *testing.T) {
 		stateUpdate := &core.StateUpdate{BlockHash: h1}
 		expectedErr := ErrIncompatibleBlockAndStateUpdate{"block hashes do not match"}
-		assert.Equal(t, chain.VerifyBlock(mainnetBlock1, stateUpdate), expectedErr)
+		assert.Equal(t, chain.SanityCheckNewHeight(mainnetBlock1, stateUpdate), expectedErr)
```

Comment:
Wherever we are using `SanityCheckNewHeight()` for testing it should be separated into a separate set of tests called: `TestSanityCheckNewHeight()`.

Reply:
done